DakshChan/IBS
# [DEV-007] Migrate Interview Timetable Pages to MUI

## [DEV-007] interviewTimetableMUI



### Checklist
- [x] Is there sufficient code commenting?
- [x] Have corresponding changes been made to documentation?
- [ ] If possible, have you written new tests for core changes?

### Changes
Explain the changes made at a higher level

- Use Flexy's components (in addition to MUI components) to migrate existing interview pages for both TAs and students from bare `JSX/HTML` code.

### Testing

0. Use the given `Makefile` to run `make build` and `make run` (given that `.env` file on `/frontend` has `PORT=3000` and `REACT_APP_API_URL=/api`) to build and run the Docker containers.
1. Then, create a course, task, and register a TA into the course.
2. Login as a TA, go to the task's `Book` section, then schedule an interview. (This page was what was modified)
3. Register a student, login as a student, and go to the task's `Book` section, then book interviews, cancel interviews, change interviews, etc. (All functionalities should behave as intended)

### References
- Include any references you may have used or any link to documentation related to this issue
    - [Flexy Main Demo](https://demos.wrappixel.com/premium-admin-templates/react/flexy-react/main/#/dashboards/dashboard1)
